scripts/startLocalFunctionsTestnet.js

```diff
@@ -0,0 +1,54 @@
+const process = require("process")
+const path = require("path")
+const fs = require("fs")
+const { startLocalFunctionsTestnet } = require("@chainlink/functions-toolkit")
+const { utils, Wallet } = require("ethers")
+// Loads environment variables from .env.enc file (if it exists)
+require("@chainlink/env-enc").config("../.env.enc")
+;(async () => {
+  const requestConfigPath = path.join(process.cwd(), "Functions-request-config.js") // @dev Update this to point to your desired request config file
+
+  const localFunctionsTestnetInfo = await startLocalFunctionsTestnet(
+    requestConfigPath,
+    {
+      logging: {
+        debug: false,
+        verbose: false,
+        quiet: true, // Set this to `false` to see logs from the local testnet
+      },
+    } // Ganache server options (optional)
+  )
+
+  console.log("\nFunctionsRouter Contract Address:", localFunctionsTestnetInfo.functionsRouterContract.address)
+  console.log("DON ID:".padStart(33, " "), localFunctionsTestnetInfo.donId)
+  console.log("Mock LINK Token Contract Address:", localFunctionsTestnetInfo.linkTokenContract.address, "\n")
```

Comment:
havent used this in a while, so worth checking but may be a more elegant way to log this data without needing padding
https://developer.mozilla.org/en-US/docs/Web/API/console/table

Reply:
I forgot about console.table. I have never had a reason to use it up until now, but it does make sense here. Thanks!